internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -852,6 +852,28 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 								string(containerservice.LicenseTypeWindowsServer),
 							}, false),
 						},
+						"gmsa_profile": {
```

Comment:
`_profile` doesn't add any informational value, so i'd suggest removing it.
```suggestion
						"gmsa": {
```